pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_uploader_tool.adoc

```diff
@@ -0,0 +1,66 @@
+[id="uploader_tool"]
+
+= Uploading content with the Uploader tool
+
+Many of the operations available in the Pantheon user interface can also be performed using commands in the command line interface.
+
+Before you can use the uploader tool, you need to install the tool and make sure that the Pantheon2.yml configuration script is present.
+
+.To install the uploader tool:
+. Install the uploader script with `setup.sh`.
++
+[options="nowrap" subs="normal"]
+----
+curl -S https://raw.githubusercontent.com/redhataccess/pantheon/master/uploader/setup.sh | sh -
+----
+. Enter ‘pantheon’ to confirm it was properly installed.
++
+[options="nowrap" subs="normal"]
+----
+$ pantheon
+----
+
+The following output appears:
+[options="nowrap" subs="normal"]
+----
+usage: pantheon [-h] [--server SERVER] [--repository REPOSITORY] [--user USER]
+[--password PASSWORD] [--directory DIRECTORY] [--verbose]
+[--dry] [--sandbox] [--sample]
+push [push ...]
+pantheon: error: the following arguments are required: push
+----
+
+.To configure the repository as a content source for Pantheon:
```

Comment:
Should this and the procedure below be separate procedure modules?

Happy to change this into an assembly with three procedures if that's easier.

Reply:
Agree with @adahms that these should be different procedures:

- Installing and configuring the uploader 
- Uploading content using the uploader

However, I don't think an assembly is required at this point.

Reply:
@theashiot @adahms I agree, that's what I suggested at first for this task. Configuration here is not just about the uploader tool, it's about configuring your repo to import content into Pantheon. Now that I've written a draft, including the example file, I suggest having the following modules:

1. `con_pantheon_yaml_file.adoc`: The Pantheon configuration file
1. `proc_config_repo.adoc`: Configuring a repo as a content source for Pantheon
1. `proc_installing_uploader.adoc`: Installing the Uploader tool
1. `proc_uploader_tool.adoc`:  Uploading content using the Uploader tool
1. `proc_importing_content.adoc`: Importing content from a Git repository

Reply:
I've added:
* proc_config_repo.adoc
* proc_installing_uploader.adoc

I've edited con_pantheon_yaml_file.adoc and proc_uploader_tool.adoc.

I don't have the info for importing content from a Git repo.